server/js/sockets.js

```diff
@@ -201,6 +201,12 @@ sockets.init = function socketInit(server) {
       );
     });
 
+    socket.on('send-message', (message) => {
+      const payload = JSON.stringify({ message });
+      mqttClient.publish('/v3/camera/message', payload);
+      socket.emit('send-message-received');
```

Comment:
Yeah I was kind of thinking this as I wrote it. I think my justification was that it's good to get *some* kind of feedback when you press the send button, and it was a decision between just showing "sent!" and assuming everything went fine or having some kind of checks to make sure that's not a lie. I'd be happy to remove it if it's just unnecessary complexity

Reply:
I think there isn't much purpose in checking the socket.io connection as there isn't really a concern there with dropping packets. A "message received" response from the camera module may be quite useful though since the radio protocol may have some issues with packet collisions.

Reply:
Yeah sounds good. Now that a month has passed and I'm no longer biased from the amount of time I spent trying to figure out React writing this, I'll go remove it :joy: